rezerwacje-app/src/main/java/org/niewidoczniakademicy/rezerwacje/service/converter/RecruitmentPeriodConverter.java

```diff
@@ -1,18 +1,34 @@
 package org.niewidoczniakademicy.rezerwacje.service.converter;
 
+import lombok.AllArgsConstructor;
+import org.niewidoczniakademicy.rezerwacje.model.database.Recruitment;
 import org.niewidoczniakademicy.rezerwacje.model.database.RecruitmentPeriod;
 import org.niewidoczniakademicy.rezerwacje.model.rest.recruitmentperiod.AddRecruitmentPeriodRequest;
+import org.niewidoczniakademicy.rezerwacje.service.exception.RecruitmentNotFoundException;
+import org.niewidoczniakademicy.rezerwacje.service.repository.RecruitmentRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
+@AllArgsConstructor(onConstructor = @__(@Autowired))
 public final class RecruitmentPeriodConverter
         implements GenericConverter<AddRecruitmentPeriodRequest, RecruitmentPeriod> {
 
+    private final RecruitmentRepository recruitmentRepository;
+
     @Override
     public RecruitmentPeriod createFrom(final AddRecruitmentPeriodRequest dto) {
+        Long recruitmentId = dto.getRecruitmentId();
+        Recruitment recruitment = recruitmentRepository
```

Comment:
Można to wydzielić do jakieś jednej metody w serwisie, bo to sie powtarza kilka razy

Reply:
Chciałem tu podpiąć RecruitmentService ale się tworzy cykliczna zależność, więc musi chyba zostać tak jak jest